processor/src/system/mod.rs

```diff
@@ -198,7 +203,7 @@ impl System {
     /// for SYSCALLs this remains set to the hash of the last invoked function.
     pub fn start_syscall(&mut self) {
         debug_assert!(!self.in_syscall, "already in syscall");
-        self.ctx = 0;
+        self.ctx = 0.into();
```

Comment:
is there a reason to do `0.into()` rather than `ContextId::root()` here?

Reply:
Nope; fixed